src/lib/config.ts

```diff
@@ -73,7 +73,6 @@ class default_config {
      * They consist of key sequences mapped to ex commands.
      */
     ignoremaps = {
-        "<S-Insert>": "mode normal",
```

Comment:
As below/above - please retain this as it is from Vimperator.

Reply:
alright, I'll remove all bindings